src/curated-corpus/components/ProspectListCard/ProspectListCard.tsx

```diff
@@ -68,12 +75,12 @@ export const ProspectListCard: React.FC<ProspectListCardProps> = (
                 : '/placeholders/collectionSmall.svg'
             }
             alt={prospect.title ?? 'No title supplied'}
-            className={classes.image}
+            sx={{ borderRadius: 4, border: '1px solid lightgray' }}
```

Comment:
Could we we use one of the app palette colours here instead of the named colour? If using inline styling with the `sx` prop, I've defaulted to using `curationPalette.lightGrey` in cases like this one.